torch/csrc/jit/codegen/cuda/utils.cpp

```diff
@@ -185,27 +228,18 @@ bool isDebugDumpEnabled(DebugDumpOption option) {
   return dump_options.at(option);
 }
 
-bool useFallback() {
-  const char* disable_fb_env = getenv("PYTORCH_NVFUSER_DISABLE_FALLBACK");
-  return !(disable_fb_env ? atoi(disable_fb_env) : false);
-}
-
-bool disableRNGUnrolling() {
-  const char* disable_rng_unroll = getenv("PYTORCH_NVFUSER_DISABLE_RNG_UNROLL");
-  return disable_rng_unroll ? atoi(disable_rng_unroll) : false;
+bool isDisabled(DisableOption option) {
+  const static auto options = parseDisableOptions();
+  return options.at(option);
 }
 
-bool disableIndexHoisting() {
-  const static char* disable_index_hoist =
-      getenv("PYTORCH_NVFUSER_DISABLE_INDEX_HOIST");
-  return disable_index_hoist ? atoi(disable_index_hoist) : false;
-}
+bool useFallback() {
+  // Keep this env var for compatibility
+  const char* disable_fb_env = getenv("PYTORCH_NVFUSER_DISABLE_FALLBACK");
```

Comment:
Nitpick, we can throw a warning here for deprecation, if user are still using `disable_fb_env`. Something like
```TORCH_WARN_ONCE("env variable PYTORCH_NVFUSER_DISABLE_FALLBACK is deprecated, use PYTORCH_NVFUSER_DISABLE=\"fallback\" instead")```;